domain/data-access/form-repository.ts

```diff
@@ -163,7 +163,13 @@ const deleteFormInstance = async (formTtl: string, instanceUri: string) => {
   await query(q);
 };
 
-const getFormInstanceCount = async (targetType: string) => {
+const getFormInstanceCount = async (
+  targetType: string,
+  options?: { limit?: number; offset?: number; sort?: string; filter?: string },
```

Comment:
most of these are not strictly necessary here but i guess it's easy to just pass it along as one thing in both functions